roles/vm_only_provision_eb_sr/templates/serviceregistry.sql.j2

```diff
@@ -577,7 +575,6 @@ INSERT INTO `janus__metadata` (`connectionRevisionId`, `key`, `value`, `created`
 INSERT INTO `janus__metadata` (`connectionRevisionId`, `key`, `value`, `created`, `ip`) VALUES (7,'redirect.sign','','2012-06-13T08:02:29+02:00','172.18.5.1');
 {% if ( not minimal_install ) %}
 INSERT INTO `janus__metadata` (`connectionRevisionId`, `key`, `value`, `created`, `ip`) VALUES (8,'AssertionConsumerService:0:Binding','urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST','2012-06-19T09:44:38+02:00','172.18.5.1');
-INSERT INTO `janus__metadata` (`connectionRevisionId`, `key`, `value`, `created`, `ip`) VALUES (8,'AssertionConsumerService:0:Location','https://grouper.{{ base_domain }}/grouper/Shibboleth.sso/SAML2/POST','2012-06-19T09:44:38+02:00','172.18.5.1');
 INSERT INTO `janus__metadata` (`connectionRevisionId`, `key`, `value`, `created`, `ip`) VALUES (8,'contacts:0:contactType','technical','2012-06-19T09:44:38+02:00','172.18.5.1');
 INSERT INTO `janus__metadata` (`connectionRevisionId`, `key`, `value`, `created`, `ip`) VALUES (8,'contacts:0:emailAddress','','2012-06-19T09:44:38+02:00','172.18.5.1');
```

Comment:
These changes only remove half the data related to the grouper entity, creating foreign key problems (and cruft).